### PR TITLE
Add changelog for 19.17

### PR DESCRIPTION
**Pull request by vitlibar**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Non-significant (changelog entry is not needed)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+## ClickHouse release v19.17.6.36, 2019-12-27
+
+### Bug Fix
+* Fixed potential buffer overflow in decompress. Malicious user can pass fabricated compressed data that could cause read after buffer. This issue was found by Eldar Zaitov from Yandex information security team. [#8404](https://github.com/ClickHouse/ClickHouse/pull/8404) ([alexey-milovidov](https://github.com/alexey-milovidov))
+* Fixed possible server crash (`std::terminate`) when the server cannot send or write data in JSON or XML format with values of String data type (that require UTF-8 validation) or when compressing result data with Brotli algorithm or in some other rare cases. [#8384](https://github.com/ClickHouse/ClickHouse/pull/8384) ([alexey-milovidov](https://github.com/alexey-milovidov))
+* Fixed dictionary with source from a clickhouse view [#8351](https://github.com/ClickHouse/ClickHouse/pull/8351) ([Nikolai Kochetov](https://github.com/KochetovNicolai))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Not descriptive enough.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+* Fixed checking if a client host is allowed by host_regexp specified in users.xml. [#8241](https://github.com/ClickHouse/ClickHouse/pull/8241), [#8342](https://github.com/ClickHouse/ClickHouse/pull/8342) ([Vitaly Baranov](https://github.com/vitlibar))
+* `RENAME TABLE` now moves data for a distributed tables now. [#8306](https://github.com/ClickHouse/ClickHouse/pull/8306) ([tavplubix](https://github.com/tavplubix))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Not descriptive enough.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+* `range_hashed` external dictionaries now allow ranges of arbitrary numeric types. ([alesapin](https://github.com/alesapin))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Not descriptive enough. It was allowed long time ago. This is just for DDL queries.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,38 @@
...
+* Fixed `DROP DICTIONARY IF EXISTS db.dict`, now it doesn't throw exception if `db` doesn't exist. [#8185](https://github.com/ClickHouse/ClickHouse/pull/8185) ([Vitaly Baranov](https://github.com/vitlibar))
+* If a table wasn't completely dropped because of server crash, the server will try to restore and load it [#8176](https://github.com/ClickHouse/ClickHouse/pull/8176) ([tavplubix](https://github.com/tavplubix))
+* Fixed a trivial count query for a distributed table if there are more than two shard local table. [#8164](https://github.com/ClickHouse/ClickHouse/pull/8164) ([小路](https://github.com/nicelulu))
+* Fixed bug that lead to a data race in DB::BlockStreamProfileInfo::calculateRowsBeforeLimit() [#8143](https://github.com/ClickHouse/ClickHouse/pull/8143) ([Alexander Kazakov](https://github.com/Akazz))
+* Fixed behavior with ALTER MOVE ran immediately after merge finish moves superpart of specified. [#8104](https://github.com/ClickHouse/ClickHouse/pull/8104) ([Vladimir Chebotarev](https://github.com/excitoon))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: superpart - is not obvious term.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,3 +1,38 @@
...
+* Now dictionaries support string as expression; default string values are fixed. [#8098](https://github.com/ClickHouse/ClickHouse/pull/8098) ([alesapin](https://github.com/alesapin))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Not descriptive enough for user.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,38 @@
...
+* `clickhouse-copier`: Refine the definition of ZXid according to the ZooKeeper Programmer's Guide. [#8088](https://github.com/ClickHouse/ClickHouse/pull/8088) ([Ding Xiang Fei](https://github.com/dingxiangfei2009))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: What bug was fixed?

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,38 @@
...
+* Fixed interpreting temporary tables as bits of query after the failure `Too many simultaneous
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Don't understand at all. Need rewrite.

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,3 +1,38 @@
...
+queries` and others. [#8084](https://github.com/ClickHouse/ClickHouse/pull/8084) ([Azat Khuzhin](https://github.com/azat))
+* Avoid null dereference after "Unknown packet X from server" [#8071](https://github.com/ClickHouse/ClickHouse/pull/8071) ([Azat Khuzhin](https://github.com/azat))
+* Restore support of all ICU locales, add the ability to apply collations for constant expressions and add language name to system.collations table. [#8051](https://github.com/ClickHouse/ClickHouse/pull/8051) ([alesapin](https://github.com/alesapin))
+* Number of streams for read from `StorageFile` and `StorageHDFS` is now limited, to avoid exceeding the memory limit. [#7981](https://github.com/ClickHouse/ClickHouse/pull/7981) ([alesapin](https://github.com/alesapin))
+* Fixex `CHECK TABLE` query for `*MergeTree` tables without key. [#7979](https://github.com/ClickHouse/ClickHouse/pull/7979) ([alesapin](https://github.com/alesapin))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: Typo.

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,3 +1,38 @@
...
+* Remove mutation number from a part name by default. [#8250](https://github.com/ClickHouse/ClickHouse/pull/8250) ([alesapin](https://github.com/alesapin))
```
Comment (alexey-milovidov) [inline CHANGELOG.md]: For what purpose?

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
Almost Ok.

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,3 +1,38 @@
...
+* Avoid null dereference after "Unknown packet X from server" [#8071](https://github.com/ClickHouse/ClickHouse/pull/8071) ([Azat Khuzhin](https://github.com/azat))
+* Restore support of all ICU locales, add the ability to apply collations for constant expressions and add language name to system.collations table. [#8051](https://github.com/ClickHouse/ClickHouse/pull/8051) ([alesapin](https://github.com/alesapin))
+* Number of streams for read from `StorageFile` and `StorageHDFS` is now limited, to avoid exceeding the memory limit. [#7981](https://github.com/ClickHouse/ClickHouse/pull/7981) ([alesapin](https://github.com/alesapin))
+* Fixed `CHECK TABLE` query for `*MergeTree` tables without key. [#7979](https://github.com/ClickHouse/ClickHouse/pull/7979) ([alesapin](https://github.com/alesapin))
+* Removed the mutation number from a part name in case there were no mutations. This remove fixes compatibility with old versions. [#8250](https://github.com/ClickHouse/ClickHouse/pull/8250) ([alesapin](https://github.com/alesapin))
```
Comment (filimonov) [inline CHANGELOG.md]: Remove fixes?

Comment (vitlibar) [reply CHANGELOG.md]: I've changed that to `This removing improved the compatibility with older versions.`

**Pull request merged**